src/core/tests/type_prop/paged_attention.cpp

```diff
@@ -126,5 +126,98 @@ TEST(type_prop, paged_attention_static_16_inputs_eviction_per_token) {
     EXPECT_EQ(op->get_output_partial_shape(0), (PartialShape{3, 4}));
 }
 
+TEST(type_prop, paged_attention_dynamic_ranks_and_types) {
+    using namespace ov::op;
+    const auto dyn = PartialShape::dynamic();
+
+    const auto query = std::make_shared<v0::Parameter>(element::dynamic, dyn);
+    const auto key = std::make_shared<v0::Parameter>(element::dynamic, dyn);
+    const auto value = std::make_shared<v0::Parameter>(element::dynamic, dyn);
+    const auto key_cache = std::make_shared<v0::Parameter>(element::dynamic, dyn);
+    const auto value_cache = std::make_shared<v0::Parameter>(element::dynamic, dyn);
+    const auto past_lens = std::make_shared<v0::Parameter>(element::dynamic, dyn);
+    const auto subsequence_begins = std::make_shared<v0::Parameter>(element::dynamic, dyn);
+    const auto block_indices = std::make_shared<v0::Parameter>(element::dynamic, dyn);
+    const auto block_indices_begins = std::make_shared<v0::Parameter>(element::dynamic, dyn);
+    const auto scale = std::make_shared<v0::Parameter>(element::dynamic, dyn);
+    const auto sliding_window = std::make_shared<v0::Parameter>(element::dynamic, dyn);
+    const auto alibi_slopes = std::make_shared<v0::Parameter>(element::dynamic, dyn);
+    const auto max_context_len = std::make_shared<v0::Parameter>(element::dynamic, dyn);
+
+    ov::OutputVector args = {query,
+                             key,
+                             value,
+                             key_cache,
+                             value_cache,
+                             past_lens,
+                             subsequence_begins,
+                             block_indices,
+                             block_indices_begins,
+                             scale,
+                             sliding_window,
+                             alibi_slopes,
+                             max_context_len};
```

Comment:
```suggestion
    define single
    ov::OutputVector args(std::make_shared<v0::Parameter>(element::dynamic, dyn), N);
```
Will it work to make N same parameters?

Reply:
I would like to preserve the input names